src/screens/Ticketing/Tickets/Tabs.tsx

```diff
@@ -35,19 +38,21 @@ export const BuyTickets: React.FC<Props> = ({navigation}) => {
       ) : (
         <View style={{flex: 1}} />
       )}
-      <View style={{padding: theme.spacings.medium}}>
-        <Button
-          mode="primary"
-          text={t(TicketsTexts.buyTicketsTab.button.text)}
-          accessibilityHint={t(TicketsTexts.buyTicketsTab.button.a11yHint)}
-          onPress={() =>
-            navigation.navigate('TicketPurchase', {
-              screen: 'PurchaseOverview',
-              params: {},
-            })
-          }
-        />
-      </View>
+      {isInitialized && !!abtCustomerId && (
```

Comment:
Could `useAuthState` return a field `isSignedIn` (or something like it) which could be used instead? When I am reading this I am not sure if `isInitialized && !!abtCustomerId` actually is the equivalent as "isSignedIn" or if it actually is something else. Doing this could also prevent exposing fields like `isInitialized` from the auth hook, hiding more of the auth complexity from the other components.

Reply:
We could probably talk about naming, but there is a difference between the two. Right now we're signing in automatically, but in the future we probably will not. So the `isInitialized` boolean signifies that we've initialized the auth-connection and attempted to fetch any existing users. In other words, when `isInitialized` is true, and there is no users signed in, you're free to attempt a sign in if you want.

The `abtCustomerId` is an id which is generated in the id-based backend after a user is signed in Firebase. This happens asynchronously, and might not be available right after a sign in. But we're unable to purchase or fetch any tickets without it, so we need to check its availability. Also we need to be able to retrieve the id itself. 

So, I'm not sure if a general `isSignedIn` from context will solve the issue in full. But I could distill it down further up in this component?

Reply:
I could maybe return a `isSignedIn` boolean depending on if `abtCustomerId` is set. But technically you are "signed in" to Firebase auth before `abtCustomerId` is set. And if generating that id is slow or fails, then `isSignedIn` would be false, even if it technically should be true. You do have a Firebase auth user which can be signed out, or do other things with.

Reply:
The suggestion `isSignedIn` was based on my interpretation of `isInitialized && !!abtCustomerId`, which I now understand has more nuances that just being signed in. But I think my misinterpration gives even more incentive to find a better named field for the logic represented by `isInitialized && !!abtCustomerId`, and maybe with some jsdoc.

Reply:
Sure, added some more comments and better naming for `isInitialized`